cmd/iso8583/describe.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/moov-io/iso8583"
+	"github.com/moov-io/iso8583/describe"
+)
+
+var specs = map[string]*iso8583.MessageSpec{
+	"87": iso8583.Spec87,
```

Comment:
Is this here so we could have `-spec 91` to change which one is used?

Reply:
right. Hope that we can add more specs :)
I'm also going to add `--spec-file xxx.json` to load spec from JSON.